### PR TITLE
Revisions to docs based on new learnings

### PR DESCRIPTION
**Pull request by ianjennings**

### CODE REVIEW

File: guide/concepts/chats/search.md [hunk 1]
```diff
@@ -43,3 +43,6 @@ chat.search({
     end: '123123123133'
 });
 ```
+
+Search will stop after looking through ```1,0000``` events and then emit the ```$.search.pause``` event.
```
Comment (ajb413) [inline guide/concepts/chats/search.md]: inline code blocks can also be placed with just single back ticks. no need for triple.

File: guide/concepts/events/payload.md [hunk 2]
```diff
@@ -32,21 +32,21 @@ ChatEngine.connect('ian', {fullName: 'Ian Jennings'});
...
+// connect with UUID 'alex'
+ChatEngine.connect('alex');
 
 // when we received a 'like' event on global chat
 ChatEngine.global.on('like', (payload) => {
 
-    // if that event matches 'emily'
-    if(payload.data.who == 'emily') {
+    // if that event matches 'alex'
+    if(payload.data.who == 'alex') {
```
Comment (ajb413) [inline guide/concepts/events/payload.md]: `if (payload.data.who === 'alex') {`

File: guide/concepts/me/me.md [hunk 3]
```diff
@@ -33,7 +33,7 @@ In order to give every user a unique name, let's create a function that returns
 ```js
 const getUsername = () => {
 
-  const animals = ['pigeon', 'seagull', 'bat', 'owl', 'sparrows', 'robin', 'bluebird', 'cardinal', 'hawk', 'fish', 'shrimp', 'frog', 'whale', 'shark', 'eel', 'seal', 'lobster', 'octopus', 'mole', 'shrew', 'rabbit', 'chipmunk', 'armadillo', 'dog', 'cat', 'lynx', 'mouse', 'lion', 'moose', 'horse', 'deer', 'raccoon', 'zebra', 'goat', 'cow', 'pig', 'tiger', 'wolf', 'pony', 'antelope', 'buffalo', 'camel', 'donkey', 'elk', 'fox', 'monkey', 'gazelle', 'impala', 'jaguar', 'leopard', 'lemur', 'yak', 'elephant', 'giraffe', 'hippopotamus', 'rhinoceros', 'grizzlybear'];
+  const animals = ['pigeon', 'seagull', 'bat', 'owl', 'sparrows', 'robin', 'bluebird', 'cardinal'];
```
Comment (ajb413) [inline guide/concepts/me/me.md]: `sparrows` is the only plural

File: guide/concepts/chats/search.md (hunk 1 above)
Comment (ianjennings) [reply guide/concepts/chats/search.md]: more = better

File: guide/concepts/events/payload.md (hunk 2 above)
Comment (ianjennings) [reply guide/concepts/events/payload.md]: more === better

File: guide/concepts/me/me.md (hunk 3 above)
Comment (ianjennings) [reply guide/concepts/me/me.md]: 👍

**Pull request merged**